readme.md

```diff
@@ -111,6 +111,12 @@ function listItem(n: number) {
 <ul>
     {[1, 2].map(listItem)}
 </ul>
+
+function Button(props: any | undefined, children: any | undefined) {
+    return <button type="button" class="original-class" {...props}>{children}</button>;
+}
+    
+<Button>Button Text</Button>
 ```
```

Comment:
You are allowed to document this feature even further, maybe add a section to [custom elements](https://github.com/nicojs/typed-html#custom-elements)